scripts/dev/generate_jdd_schema/modify_schema.py

```diff
@@ -24,13 +114,11 @@ def change_version(schema):
     loc['default'] = "${CMAKE_VERSION_MAJOR}.${CMAKE_VERSION_MINOR}"
     loc['type'] = "string"
 
-
```

Comment:
I think you should add another blank line here, so does PEP8  :)
https://www.python.org/dev/peps/pep-0008/#blank-lines